client/src/components/projects/Projects.vue

```diff
@@ -1,29 +1,34 @@
 <template>
-  <div class="columns">
-    <div class="column is-one-quarter">FILTER</div>
-    <div class="column">
+  <div class="columns projects">
+    <div class="column is-one-quarter">
+      <div class="filters"/>
       <div>
-        <a
-          class="button is-small"
-          @click="newProject()">
-          New project
-        </a>
+        <ProjectFilter
+          :customers="customers"
```

Comment:
No, ProjectFilter should not get any data from parent, or even child -> parent 
Use store!

Reply:
done